### DISCUSSION TITLE
How can I detect a webcam(stream) and make the output the same length as the original video?

### DISCUSSION THREAD

**Discussion by 7icky** [ultralytics/yolov5#4172] [Q&A]
Hello there,
I tried to use the detect.py to do the objection ditection of an online video（stream），the orignal video includes around 1300 frames. However, I only got a really short video with 110 frames after I run the detect.py with the source = 'https://stream7.iqilu.com/10339/upload_transcode/202002/18/20200218114723HDu3hhxqIT.mp4' and the weights = 'yolov5m.pt'
, which I think it meant to be 325 frames cos the code will cap every 4 frame in the Class "LoadStream". I was wondering if it is possible to get the same length output video as the input one by changing some parameter? or is there any other part of code triming the size of the output?

**Comment by glenn-jocher** [comment]
@7icky good news 😃! Your original issue may now be fixed ✅ in master. To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload with `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
